### DISCUSSION TITLE
repeated decimal

### DISCUSSION THREAD

**Discussion by yukulele** [Qalculate/libqalculate#632] [Q&A] [answered]
is there a way to show result as repeated decimal?

```shell
> qalc 7/3
7 / 3 = 2.'3'

> qalc 3/7
3 / 7 = 0.'428571'

> 61/495
61 / 495 = 0.1'23'
```

**Comment by hanna-kn** [comment, marked as answer]
```
> help repeating decimals

repeating decimals (repdeci)
If activated, 1/6 is displayed as '0.1 666...', otherwise as '0.166667'.
(0* = off, 1 = on)

> set repdeci
> 7/3

  7 / 3 = 2.333…

> 3/7

  3 / 7 = 0.428571 428571…

> 61/495

  61 / 495 = 0.123 23…

```

**Reply by yukulele** [reply]
Thank! 
It's work for `Qalc`!

How can I configure **Qalculate** to set `repdeci`?

**Reply by hanna-kn** [reply]
In qalculate-gtk: Mode➞Numerical Display➞Indicate Repeating Decimals
In qalculate-qt: Preferences➞Numbers & Operators➞Indicate repeating decimals

Alternatively, qalc commands can also be used in the GUIs (in the expression field, with `/` prepended, e.g. `/set repdeci`).
